CONTRIBUTING.md

```diff
@@ -112,26 +112,11 @@ To create an issue for adding a new component to the repo, please observe the fo
         - For PatternFly 4 components, direction will be given as to which package it will go in.  Generally, we are recommending most extension components go in individual packages.  Several smaller common components may end up being grouped together.  In the future, a generator for extension component packages will be created.
     - Extension components do not undergo the same rigorous design or coding review process as core PatternFly components.  If enough members of the community find them useful, we will work to move them into our core PatternFly system by starting the design process for the idea.
 
-### Contributing Components
-
-Components that are ready to be contributed can be found on the project board in the backlog column. They are identified with the text “Component” in the issue title.
```

Comment:
I think we still need something here about how to contribute a component.

Reply:
I think most contributors already know to just copy+paste current components.